dbms/src/Common/MyTime.cpp

```diff
@@ -745,7 +745,7 @@ inline bool numberToDateTime(Int64 number, MyDateTime & result, bool allowZeroDa
 }
 
 // returns frac, overflow, matched. eg., "999" fsp=2 will overflow.
-std::tuple<UInt32, bool, bool> parseFrac(const String & str, int8_t fsp)
+std::tuple<UInt32, bool, bool> parseFrac(const std::string_view str, int8_t fsp)
```

Comment:
The caller don't need to be updated after the input argument type is changed?

Reply:
In this case, `const String &` and `std::string_view` have the same effect. The caller won't change the original string buffer.